net.sourceforge.vrapper.core.tests/src/net/sourceforge/vrapper/core/tests/cases/SwitchBufferCommandTests.java

```diff
@@ -0,0 +1,208 @@
+package net.sourceforge.vrapper.core.tests.cases;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sourceforge.vrapper.core.tests.utils.VimTestCase;
+import net.sourceforge.vrapper.platform.Buffer;
+import net.sourceforge.vrapper.vim.commands.CommandExecutionException;
+import net.sourceforge.vrapper.vim.commands.SwitchBufferCommand;
+import net.sourceforge.vrapper.vim.modes.NormalMode;
+
+/**
+ * Test case for {@link SwitchBufferCommand}.
+ *
+ */
+public class SwitchBufferCommandTests extends VimTestCase {
+
+    @Before
+    public void activateNormalMode() {
+        adaptor.changeModeSafely(NormalMode.NAME);
+    }
+
+    /**
+     * Test that ":buffer 2" switch to buffer whose ID is 2
```

Comment:
Typo (here and in the following comments), and it could maybe use some rewording:
```suggestion
     * Test that ":buffer 2" switches to buffer with ID 2
```